docs/quickstart/docker-compose.yml

```diff
@@ -68,9 +68,9 @@ services:
                        cub sr-ready schema-registry 8081 20 && \
                        echo Waiting a few seconds for topic creation to finish... && \
                        sleep 2 && \
-                       java -jar /usr/share/java/ksql-examples/ksql-examples-4.0.0-SNAPSHOT-standalone.jar
+                       java -jar /usr/share/java/ksql-examples/ksql-examples-0.1-SNAPSHOT-standalone.jar
```

Comment:
these changes need reverting. I guess they were accidentally pulled across when you switched branches

Reply:
Yes, you're absolutely right.
Obviously, all that merging local mvn installing and so on completely confused me, git or me and and git or whatsoever.
Eventually overlooked completely that 15 files were changed and not 2 as planned.
Sorry for the inconvenience.